tests/worktree/reflog.c

```diff
@@ -22,6 +22,32 @@ void test_worktree_reflog__cleanup(void)
 	cleanup_fixture_worktree(&fixture);
 }
 
+void test_worktree_reflog__read_worktree_HEAD(void)
+{
+	git_reflog *reflog;
+	const git_reflog_entry *entry;
+
+	cl_git_pass(git_reflog_read(&reflog, fixture.worktree, "HEAD"));
+	cl_assert_equal_i(git_reflog_entrycount(reflog), 1);
+
+	entry = git_reflog_entry_byindex(reflog, 0);
+	cl_assert(entry != NULL);
+	cl_assert_equal_s(git_reflog_entry_message(entry), "checkout: moving from 099fabac3a9ea935598528c27f866e34089c2eff to testrepo-worktree");
```

Comment:
Our `cl_assert` tests are for `expected` followed by `actual`.  Could you please flip-flop these, eg:

    `cl_assert_equal_i(1, git_reflog_entrycount)refcount)`, and
    `cl_assert_equal_s("checkout: ...", git_reflog_entry_message(entry))`

Reply:
I just copied the test case from this file, so it's also flipped there...